### ISSUE TITLE
log(x, a) (log in any base) is turn in log(x) when I'm parsing to latex

### ISSUE THREAD

**Issue by Gabriel-Gutic** [sympy/sympy#25429]
I'm trying to print some logarithms. I tried with sympy 1.11 and 1.12 and the same issue. I don't get the base, I mean I know that, if I want to print log(x, 3), I should get log(x)/log(3) which is fine, but I'm only getting log(x)

**Comment by asmeurer** [comment]
CC @wermos @Upabjojr

**Comment by Gabriel-Gutic** [comment]
Ok, I found out a solution. I used sympy.simplify, before using sympy.latex and it seems to work

**Comment by wermos** [comment]
@Gabriel-Gutic Could you give us a minimal working example which demonstrates the bug? That would help with debugging and help me understand the issue better.

**Comment by meganly** [comment]
Printing does not work with evaluate=False
```
>>> latex(log(x,2,evaluate=False))
'\\log{\\left(x \\right)}'
```

**Comment by asmeurer** [comment]
There's actually a fix for that at https://github.com/sympy/sympy/pull/19396. 

But I wouldn't expect *anything* to work with an evaluate=False 2-arg `log`, for the reasons described at https://github.com/sympy/sympy/issues/6888#issuecomment-392141637. `log(x, a, evaluate=False)` should maybe be completely disallowed for this reason, at least until that issue is fixed.

**Comment by sylee957** [comment]
I think that the usage of 2-arg logarithm is removed after LaTeX parser had changed. The original issue came from old LaTeX parser trying to abuse that feature.

Anyway, I don't really find a problem that SymPy should not support 2-argument log eventually.

**Comment by asmeurer** [comment]
This issue predates the latex parser. The 2-argument log is a useful thing to represent. If you are working in a log that's naturally in a base other than e, you want to be able represent that directly, rather than having a bunch of extra log(b) terms floating around.

**Comment by sylee957** [comment]
I meant the same thing that we need the log with 2 base without evaluating with fraction anyway, and it won’t be more difficult than atan with 2 arguments, that we already have.
However, we need to think about how to name the function, or should we reuse the name or implementation of log, and how to make other parts of SymPy not ignore the new log representation

**Comment by asmeurer** [comment]
Yes, that's really the point of #6888. I can't really decide which way I prefer. On the one hand, the simplest way to handle the implementation is without doubt to just change the definition of `log` from `log(x)` to `log(x, base=E)`. On the other hand, this implies a backwards compatibility break for any function that handles `log` without checking the number of `args`. I can't decide if that backwards compatibility break is worth the simpler implementation (the alternative more backwards compatible implementation is to have a separate class, but this is really messy compared to just using one `log` class).
